Resources/Prototypes/_Mono/Shipyard/Expedition/duran.yml

```
@@ -13,9 +14,9 @@
 - type: vessel
   id: Duran
   parent: BaseVessel
-  name: SKR Duran
-  description: A general-purpose expeditionary vessel with a complementary kitchen.
-  price: 31005
+  name: NIKI Duran
+  description: A cheap piece of shit exped shuttle with little to no armour, but by god, does it have soul.
```

```suggestion
  description: A cheap general-purpose expeditionary shuttle with little to no armour, but by god, does it have soul.
```
Ughhh.
The shortened slang "Exped", the swearing in the shuttle description, the even less descriptive of what it has as it no longer mentions "kitchen" or "multi-purpose".